src/general.js

```diff
@@ -20,4 +20,26 @@ class MailchimpInput extends React.Component {
     }
 };
 
-export default MailchimpInput;
\ No newline at end of file
+class General extends React.Component {
+    render(){
+        return(
+            <section id="general">
+                <div id="wire" className="desktop-hide"/>
+                <div id="general-content">
+                    <div id="hackathon-name">
+                        BRICKHACK 7
+                    </div>
+                    <div id="hackathon-description">
+                        RIT's Premiere Hackathon
+                    </div>
+                    <div id="hackathon-info">
+                        Feb 20-21 | ROCHESTER INSTITUTE OF TECHNOLOGY
+                    </div>
+                    <MailchimpInput/>
```

Comment:
If we're using mailchimp input only once I think it'd be a bit cleaner to just put that in here instead of another component, there's only 10 lines of HTML